README.md

```diff
@@ -1 +1 @@
-README
+This is a test. This is a test of the outdoor warning system. This is only a test.
```

Comment:
How is this repo a test of the outdoor warning system? I think this is a reference to https://twitter.com/sfsiren